src/XcooBee/Core/Api/Bees.php

```diff
@@ -131,12 +131,15 @@ public function takeOff(array $bees, $options, $subscriptions = [], $config = []
         }';
 
         $params = [
-            'filenames'         => $options['process']['fileNames'],
             'user_reference'    => array_key_exists('userReference', $options['process'])
                 ? $options['process']['userReference']
                 : null,
         ];
 
+        if (array_key_exists('fileNames', $options['process'])) {
```

Comment:
you can add test case for this. I.e. if filenames not passed

Reply:
@mahmoudsaeed 
have we resolved this?

Reply:
@Bilal-S Not yet.